.drone.yml

```diff
@@ -30,6 +31,18 @@ pipeline:
     when:
       branch: master
 
+  warehouse-drone publish:
+    group: publish
+    image: plugins/docker
+    repo: futurelearn/warehouse-drone
```

Comment:
This repo will need to be added in Dockerhub! If you don't already have a user, create one and I'll give you admin permissions to the Futurelearn dockerhub repo.

Reply:
I made a huge typo when creating my account so now waiting on a reply for my support ticket. Hopefully will sort that out by the end of the day.

Reply:
So I still haven't heard back about my account deletion request. Would you be able to set this docker hub repo up for me @surminus?

Reply:
Yup, no problem.

Reply:
Done! https://hub.docker.com/r/futurelearn/warehouse-drone/